.markdownlintignore

```diff
@@ -0,0 +1,3 @@
+/CHANGELOG.md
+node_modules
+python/iog
```

Comment:
Why do we want to ignore python/iog ?

Reply:
Nice catch!
Because, at the time of writing this commit (a few days before I started working), I was not aware that iog was one of our projects 😅 
In the doubt, I believed that iog was a copy of the source code from an external project so I avoided linting its files.

- [ ] Now that it's clearer to me, I'll remove this line and clean iog documentation 🚀